.github/workflows/update_package_properties.yml

```diff
@@ -27,10 +30,10 @@ jobs:
         uses: actions/checkout@v2
 
       - name: Install dependencies
-        run: sudo apt install libcurl4-openssl-dev libssl-dev
+        run: sudo apt-get update && sudo apt install libcurl4-openssl-dev libssl-dev python3-testresources
 
       - name: Clone tools branch
-        run: git clone -b v0.8.1 --depth=1  https://github.com/citusdata/tools.git tools
+        run: git clone -b update_properties_fixes --depth=1  https://github.com/citusdata/tools.git tools
```

Comment:
needs to be updated

Reply:
@gurkanindibay Please update the PR description to indicate that this PR is blocked on a tools PR.

If you do not provide a link, it requires digging everytime. I do not want to spend time going through tools repo PRs and see which one actually has `update_properties_fixes` as the base branch.

Reply:
I've already added one in description https://github.com/citusdata/packaging/pull/782#issue-737431653

Reply:
> I've already added one in description [#782 (comment)](https://github.com/citusdata/packaging/pull/782#issue-737431653)

No, I did that. Please provide links in the PR description in the future.